### ISSUE TITLE
Search page improvments

### ISSUE THREAD

**Issue by pawelczak** [CeON/saos#670]
- results list - drop border from keywords,
- search form - hint for search form(text needed),
- search form - hide date description, but screen reader should be able to read it,
- move dateFrom & dateTo to one line,
- hide button clear search form.
